### DISCUSSION TITLE
Option for participants to add dates/time slots

### DISCUSSION THREAD

**Discussion by yamsu** [lukevella/rallly#1194] [Ideas]
Lovely App! Thanks for all the hard work, in getting it so far.

I would like to see an option where participants can add date &/or time options to the current poll. This would be equivalent to a poll app's 'others' field.

**Comment by lukevella** [comment]
I like this idea but think it's quite difficult to execute without overcomplicating things.

**Reply by tiptoptom** [reply]
What do you think would become too complicated, the code or the UI?
